docs/about.md

```diff
@@ -2,44 +2,139 @@
+
+:glyphicon-console: Interested in **contributing to MarkBind**? Visit the [MarkBind project on GitHub](https://github.com/MarkBind/markbind).
+
+## Contributors ✨
 
-* [**Aaron Chong Jun Hao**](https://github.com/acjh): _Project Mentor_ since Jul 2018, _Team Lead_ for Aug 2017 - Jun 2018
-* [**Ang Ze Yu**](https://www.github.com/ang-zeyu): _Project Member_ since Jan 2020
-* [**Chng Zhi Xuan**](https://github.com/Chng-Zhi-Xuan): _Project Member_ since May 2018
-* [**Chua Yun Zhi Nicholas**](https://github.com/nicholaschuayunzhi): _Project Member_ since Jan 2018
+**Mentors**:
 * [**Damith C. Rajapakse**](https://www.comp.nus.edu.sg/~damithch/): _Project Mentor_ since Aug 2016
-* [**Daryl Tan**](https://www.github.com/openorclose): _Project Member_ since May 2019
-* [**Jamos Tay**](https://github.com/jamos-tay): _Project Member_ since Aug 2018
-* [**Labayna Neil Brian Narido**](https://www.github.com/nbriannl): _Project Member_ since Jan 2020
-* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Mentor_ since Jan 2020, _Project Member_ for Jan 2019 - Aug 2019
-* [**Tan Wang Leng**](https://github.com/yamgent): Current _Team Lead_ since Jul 2018, _Project Member_ for May 2018 - Jun 2018
-* [**Tan Yuanhong**](https://www.github.com/le0tan): _Project Member_ since Jan 2020
-* [**Yash Chowdhary**](https://www.github.com/yash-chowdhary): _Project Member_ since Jan 2020
-* [**Yip Seng Yeun**](https://www.github.com/alyip98): _Project Member_ since May 2019
-
-Contributors:
-* [**Ahmed Bahajjaj**](https://www.github.com/madanalogy): _Contributor_ since Jan 2020
-* [**Daryl Chan**](https://www.github.com/dvrylc): _Contributor_ since Jan 2020
-* [**Jeremy Tan Jie Rui**](https://www.github.com/Parcly-Taxel): _Contributor_ since Jan 2020
-* [**Joshua Wong**](https://www.github.com/hcwong): _Contributor_ since Jan 2020
-* [**Phang Chun Rong**](https://www.github.com/crphang): _Contributor_ since Jan 2020
-* [**Tejas Bhuwania**](https://www.github.com/Tejas2805): _Contributor_ since Jan 2020
-
-Past Members:
-* [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Project Member_ for Jan 2018 - Aug 2018
-* [**Jia Zhixin**](https://github.com/nusjzx): _Project Member_ for May 2018 - Aug 2018
-* [**Jiang Sheng**](https://github.com/Gisonrg): _Founding Member_ and _Team Lead_ for Aug 2016 - Jul 2017
-* [**Lin Si Jie**](https://www.github.com/sijie123): _Project Member_ for Jan 2019 - Aug 2019
-* [**Lu Yang Kenneth**](https://github.com/luyangkenneth): _Project Member_ for Jan 2019 - Aug 2019
-* [**Maddi Aadyaa**](https://www.github.com/amad-person): _Project Member_ for Jan 2019 - Aug 2019
-* [**Rachael Sim Hwee Ling**](https://github.com/rachx): _Project Member_ for Jan 2018 - Dec 2018
-* [**Tan Zhen Yong**](https://www.github.com/Xenonym): _Project Member_ for Jan 2019 - Aug 2019
 
-:glyphicon-send: You can **email us** at `markbind` at `comp.nus.edu.sg`
+**Dev Team**:
```

Comment:
^ Missing the most recent batch (you guys) + some date touchups

```
* [**Ang Ze Yu**](https://www.github.com/ang-zeyu): **_Project Lead_** since Jan 2021, Member since Jan 2020
* [**Jonah Tan Jun Zi**](https://www.github.com/jonahtanjz): _Senior Dev_ since Apr 2021, _Member_ since Jan 2021
* [**Koh Rayson**](https://www.github.com/raysonkoh): _Senior Dev_ since Apr 2021, _Member_ since Jan 2021
* [**Ong Wei Xiang**](https://www.github.com/wxwxwxwx9): _Senior Dev_ since Apr 2021, _Member_ since Aug 2020
* [**Ryo Chandra Putra Aramanda**](https://www.github.com/ryoarmanda): _Senior Dev_ since Apr 2021, _Member_ since Aug 2020
* [**Hannah Chia Kai Xin**](https://www.github.com/kaixin-hc): _Senior Dev_ since Apr 2022, _Member_ since Jan 2022
* [**Jovyn Tan Li Shyan**](https://www.github.com/jovyntls): _Senior Dev_ since Apr 2022, _Member_ since Jan 2022
* [**Liu Yong Liang**](https://www.github.com/tlylt): _Senior Dev_ since Apr 2022, _Member_ since Aug 2021
* [**Ong Jun Xiong**](https://www.github.com/ong6): _Senior Dev_ since Apr 2022, _Member_ since Aug 2021
```

> My proposed plan for maintaining this page is as follows:

Sounds good 👍 

> Hi @jonahtanjz, thanks for the heads-up. Interestingly, this is an unexpected hydration issue, in which the all-contributors-cli generates a table without `<tbody>` element, which syntactically is fine but would error out for Vue due to the mismatch in HTML content. I have followed up the issue in the source repo [here](https://github.com/all-contributors/cli/pull/307#issuecomment-1115835646), will wait and see if there are any updates.

The hydration issue seems to be causing the header runtime detection to not-run as well (and the first heading to be "squeezed" against the header). Maybe we can add a `mt-3` margin to the first heading.

Looks really nice otherwise, thanks for consolidating this! @tlylt